python_modules/libraries/dagster-dask/dagster_dask_tests/hammer_job.py

```diff
@@ -0,0 +1,9 @@
+from dagster_dask import dask_executor
```

Comment:
Dagster-dask test was using a pipeline created within the toys module; moved that here so that we don't have some wacky dagster-dask import within our toys repo.